source/views/layouts/govuk_template.html.erb

```diff
@@ -110,7 +110,12 @@
             <%= yield :footer_support_links %>
 
             <div class="open-government-licence">
-              <p class="logo"><a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/" rel="license">Open Government Licence</a></p>
+              <% if content_for?(:licence_logo) %>
+                <%= yield :licence_logo %>
```

Comment:
I guess I would override it with a zero length string in my local copy of this file:

https://github.com/alphagov/govuk_elements/blob/master/lib/template-config.js

Though I could be wrong. Part of the difficulty with this kind of thing is the need to target so many templating languages?! I.e. with mustache being "logicless"